### ISSUE TITLE
Add death state

### ISSUE THREAD

**Issue by dekrain** [liam4/slw#10]
Here:
![slw - texture bug](https://cloud.githubusercontent.com/assets/19158291/19777385/482510a8-9c78-11e6-959d-671889a1cde0.png)

**Comment by towerofnix** [comment]
That's actually a death zone:

``` js
['#', class extends Tile {
  constructor(game) {
    super(game, {
      name: 'Death Zone',
      position: [0, 0],
    })
  }

  onTouch(by: Entity) {
    // TODO destroy it
    if(by instanceof Player) console.warn('you are ded')
  }
}]
```

..clearly yet to be finished.

But yeah, it could probably have a better texture.

**Comment by dekrain** [comment]
Thanks

**Comment by bates64** [comment]
> But yeah, it could probably have a better texture.

Or better, _none at all_!

**Comment by bates64** [comment]
Resolved through #12 in 34e94ff6ec5e27f4e29629206af9bea9be6e6ee4.
